tests/cypress/integration/actions_users/registration_involved/case_28_review_pipeline_feature.js

```diff
@@ -351,25 +351,23 @@ context('Review pipeline feature', () => {
         });
 
         it('Select an issue on sidebar. Issue indication has changed the color for highlighted issue', () => {
-            let index = 0;
             cy.collectIssueRegionId().then(($issueRegionList) => {
-                cy.get('.cvat-objects-sidebar-issue-item').then((sidebarIssueItems) => {
-                    for (let i = 0; i < sidebarIssueItems.length; i++) {
-                        cy.get(sidebarIssueItems[i]).trigger('mousemove').trigger('mouseover');
-                        cy.get(`#cvat_canvas_issue_region_${$issueRegionList[index]}`).should(
-                            'have.attr',
-                            'fill',
-                            'url(#cvat_issue_region_pattern_2)',
-                        );
-                        cy.get(sidebarIssueItems[i]).trigger('mouseout');
-                        cy.get(`#cvat_canvas_issue_region_${$issueRegionList[index]}`).should(
-                            'have.attr',
-                            'fill',
-                            'url(#cvat_issue_region_pattern_1)',
-                        );
-                        index++;
-                    }
-                });
+                for (let i = 0; i < $issueRegionList.length; i++) {
```

Comment:
I believe construction for .. of is more suitable here
``for (const issueRegionID of issueRegionList)``

Reply:
BTW, I looked at implementation of ``collectIssueRegionId`` command and I see the regex there does not consider negative IDs
But when new issue is created and not saved on the server yet, it has exactly negative ID. It could lead to unexpected bugs in the future
Possible solution is to change regex to ``.match(/\-?\d+$/)``
What do you think?

Reply:
> I believe construction for .. of is more suitable here

You're right. It's more convenient. Thanks.

> Possible solution is to change regex to

Right. This may come in handy in the future. It also allowed you to slightly reduce the code in the ```checkIssueRegion``` command